satellite/gracefulexit/endpoint.go

```diff
@@ -455,7 +491,7 @@ func (endpoint *Endpoint) handleFailed(ctx context.Context, pending *pendingMap,
 	}
 
 	// only increment failed if it hasn't failed before
```

Comment:
Update comment - "only increment overall failed count if piece failures is above threshold"